pattern-2/bosh-release/jobs/wso2apim/templates/ctl.erb

```diff
@@ -65,7 +65,7 @@ dpkg -s nfs-common >/dev/null 2>&1
 if [ $? -ne 0 ] ; then
   log_debug "Installing NFS Client"
   sudo apt-get update
-  sudo apt-get install nfs-common
+  sudo apt-get install nfs-common -y
```

Comment:
This is needed if we are changing the stemcell. But the stemcell in the tile.yaml is still the same.

Reply:
Shall we change the stemcell version as well?

Reply:
+1